cfpb/cfgov-refresh
# Don't allow dashes in contact phone fields

Closes https://github.com/cfpb/cfgov-refresh/issues/3214.

The [contact-phone template](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/_includes/molecules/contact-phone.html#L40) puts phone numbers into a standard phone number format, e.g. `(555) 444-6789`. So, when a user enters a number with formatting, the template adds extra formatting to it, e.g. `(333) .55-5.4444`, which looks wrong.

This PR adds a validation to ensure that entered phone numbers only contain digits. It also adds help text to help content editors understand how to enter phone numbers. Any vanity phone numbers, e.g.
(333) MY-1-CFPB can have any characters, and won't get automatically formatted.

Validations won't run until we upgrade to Wagtail 2 :umbrella:.

## Additions

- Validation to ensure phone numbers are made up of only digits
- Help text for phone number fields

## Testing

1. Wait until we upgrade to the latest version of wagtail
2. Running this branch, create a Browse Page
3. Add an Expandable unit to the Content section
4. Add a Phone block to the Expandable
5. Enter a variety of phone numbers in the fields
    - Upon saving the page, the validation should prevent saving if the phone field contains non-digit characters.

## Screenshots
<img width="758" alt="Screen Shot 2019-07-23 at 11 49 07 AM" src="https://user-images.githubusercontent.com/4295388/61726625-ecdd9400-ad3f-11e9-9f52-ff2ec7c1e1db.png">


## Notes

- Technically, this won't break anything if we merge. The help text on the page will still be helpful.
- However, the validation won't actually run until we upgrade to Wagtail 2. :x::x:

## Checklist
